cookbooks/aws-parallelcluster-config/recipes/head_node_base.rb

```diff
@@ -30,22 +30,22 @@
   network get_vpc_cidr_list
   writeable true
   options ['no_root_squash']
-end
+end unless virtualized?
```

Comment:
Why did you need to exclude docker from it? How did system tests work without this exclusion?

Reply:
Export filesystem is difficult from a docker container. I do not see a specific value to invest in test it in the docker container. What do you think?